test/cases/filesystem.sh

```diff
@@ -156,7 +156,7 @@ size = 131072000
 
 [[customizations.filesystem]]
 mountpoint = "/var/log/audit"
-size = 131072000
+size = 125 MiB
```

Comment:
```suggestion
size = "125 MiB"
```

ah, it must be a string: https://toml.io/en/v1.0.0#string